azplugins/bond.py

```diff
@@ -55,17 +55,22 @@ def __init__(self, name=None):
         hoomd.context.current.system.addCompute(self.cpp_force, self.force_name)
 
         # setup the coefficient options
-        self.required_coeffs = ['V_max','a','b']
+
+        self.required_coeffs = ['V_max','a','b','c']
```

Comment:
Does the HOOMD bond potential support default coefficients, like the pair potentials? If so, we should make `c=0` the default so that the original behavior of the class is maintained, and mark it in the documentation above.

Reply:
It does! I didn't realize, this is a very good suggestion.